modules/core/src/main/java/org/apache/ignite/cache/query/SqlFieldsQuery.java

```diff
@@ -66,7 +69,7 @@
     private boolean collocated;
 
     /** Query timeout in millis. */
-    private int timeout;
+    private int timeout = DFLT_QUERY_TIMEOUT;
```

Comment:
Also we will have some limitations regarding default timeout usage for other platform clients and ODBC. For them it is impossible to realize if a default timeout should be used if current values is 0. We can add handling of `-1` for them in this PR (similarly to `ClientCacheSqlFieldsQueryRequest`). Changing platforms code can be done later.